lib/client-assets.php

```diff
@@ -200,6 +237,14 @@ function gutenberg_register_packages_scripts( $scripts ) {
 		$dependencies = isset( $asset['dependencies'] ) ? $asset['dependencies'] : array();
 		$version      = isset( $asset['version'] ) ? $asset['version'] : $default_version;
 
+		// Register the runtime dependency for each entry point.
+		array_push( $dependencies, 'wp-runtime' );
```

Comment:
FYI, better to just use `$dependencies[] = 'wp-runtime'`. No overhead of calling a function.